NOTICE.txt

```diff
@@ -113,6 +113,52 @@ SOFTWARE.
 
 ---
 
+## @react-native-community/cookies
+
+This product contains '@react-native-community/cookies' by React Native Community.
+
+Cookie Manager for React Native
+
+* HOMEPAGE:
+  * https://github.com/react-native-community/cookies
+
+* LICENSE: MIT License
+
+Copyright (c) 2020 React Native Community
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+---
+
+## @react-native-community/masked-view
+
+This product contains '@react-native-community/masked-view' by React Native Community.
+
+React Native Masked View Library
+
+* HOMEPAGE:
+  * https://github.com/react-native-community/react-native-masked-view
+
+* LICENSE:
```

Comment:
No license

Reply:
this one is from `react-native-community` so is fine to use, no license file but it is MIT 
![image](https://user-images.githubusercontent.com/6757047/83079941-e8ae9100-a04a-11ea-923b-393ba374b8c3.png)